src/components/ui/EventsList.tsx

```diff
@@ -63,28 +68,32 @@ const EventsList: React.FC<EventsListProps> = ({ events }) => {
       (a, b) => b.sortableTime - a.sortableTime,
     );
 
-  const rows = sortedEvents.map((event) => ({
-    cells: [
-      {
-        title: <strong>{getHumanizedDateTime(event.eventTime)}</strong>,
-      },
-      {
-        title: (
-          <>
-            {event.severity !== 'info' && (
-              <>
-                <Label color={getLabelColor(event.severity)} icon={getLabelIcon(event.severity)}>
-                  {event.severity}
-                </Label>{' '}
-              </>
-            )}
-            {event.message}
-          </>
-        ),
-      },
-    ],
-    props: { event },
-  }));
+  const rows = sortedEvents.map((event) => {
+    const hostname = event.hostId ? findHostname(event.hostId, hosts) : null;
```

Comment:
IMO, it would be more comfortable if the `findHostname` handles false-like `event.hostId` as well

Reply:
Following our discussion we agreed that in that case whoever wants to use `findHostname` will need to peek into the function definition in order to see when this function returns `null`.
See here 25a9c1865a9448d45fdaed6178ad57c5876c5bc9 for the fix.